sdk/storage/Azure.Storage.DataMovement.Blobs/src/DataMovementBlobsExtensions.cs

```diff
@@ -382,6 +378,23 @@ internal static BlockBlobStageBlockOptions ToBlobStageBlockOptions(this BlockBlo
                 DestinationConditions = CreateRequestConditions(options?.DestinationConditions, overwrite),
                 SourceAuthentication = sourceAuthorization,
             };
+            if ((options?.ContentEncoding?.Preserve ?? true) &&
+                (options?.ContentDisposition?.Preserve ?? true) &&
+                (options?.ContentLanguage?.Preserve ?? true) &&
+                (options?.ContentType?.Preserve ?? true) &&
+                (options?.CacheControl?.Preserve ?? true) &&
+                (options?.AccessTier?.Preserve ?? true) &&
```

Comment:
Does Put Blob from URL always copy access tier? Or does it depend on `CopySourceBlobProperties`?

If it always copies it no matter what, I don't think we should have `AccessTier` here?

Reply:
Will address in my next PR.